enable_apis/main.tf

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
@milesmatthias Please move this into `modules/enable_apis`

Reply:
done